src/stub/language/variable_name_options.rs

```diff
@@ -49,8 +49,19 @@ impl VariableNameOptions {
         }
     }
 
+    /// Escapes a variable name if it is contained in the vector of (disallowed)
+    /// keywords. This performs a case-insensitive comparison.
     fn escape_keywords(&self, variable_name: String) -> String {
-        if self.keywords.contains(&variable_name) {
+        // This should be language dependent.
```

Comment:
Maybe a `case_insensitive_keywords` option key that defaults to false?

Reply:
Added.